osu.Game/Beatmaps/IWorkingBeatmap.cs

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
+#nullable enable
```

Comment:
This nullability enable on the interface is also raising a fair few inspections (but those ones look mostly wrong, i.e. the code is correct and the inspections are wrong due to lack of nullability annotations).

Reply:
guess it's best to remove this too..